### DISCUSSION TITLE
using this to make a custom 2 and 3 gang light switch.

### DISCUSSION THREAD

**Discussion by rchiileea** [ExperienceLovelace/ha-floorplan#140] [Ideas]
Just out of curiosity as I am complete still what I class as a noobie to hassio (only 1 month in) but could this format be used to create a actual working switch too (by that I mean have a custom 3 gang or 4 gang switch svg image and actually have presses mapped to buttons of the switch if each button was mapped right?

**Comment by OzGav** [comment]
If I understand you correctly then yes. Any part of the image can be mapped to do whatever you need. Have a look here where @pkozul made a TV remote out of floorplan https://experiencelovelace.github.io/ha-floorplan/docs/example-remote/

**Reply by rchiileea** [reply]
Thanks to the response, ill have to deep dive into it once switches are here and added
